[2024-11-04T05:07:25.528+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-04T05:07:25.538+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_new.pivot_data_test manual__2024-11-04T05:06:12.986905+00:00 [queued]>
[2024-11-04T05:07:25.619+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_new.pivot_data_test manual__2024-11-04T05:06:12.986905+00:00 [queued]>
[2024-11-04T05:07:25.619+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-04T05:07:25.624+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): pivot_data_test> on 2024-11-04 05:06:12.986905+00:00
[2024-11-04T05:07:25.628+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=300) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-04T05:07:25.630+0000] {standard_task_runner.py:63} INFO - Started process 309 to run task
[2024-11-04T05:07:25.630+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'datapipeline_new', 'pivot_data_test', 'manual__2024-11-04T05:06:12.986905+00:00', '--job-id', '762', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmplyg3rk9f']
[2024-11-04T05:07:25.631+0000] {standard_task_runner.py:91} INFO - Job 762: Subtask pivot_data_test
[2024-11-04T05:07:25.649+0000] {task_command.py:426} INFO - Running <TaskInstance: datapipeline_new.pivot_data_test manual__2024-11-04T05:06:12.986905+00:00 [running]> on host 7ffb0854de89
[2024-11-04T05:07:25.680+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='MLOPS' AIRFLOW_CTX_DAG_ID='datapipeline_new' AIRFLOW_CTX_TASK_ID='pivot_data_test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T05:06:12.986905+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-04T05:06:12.986905+00:00'
[2024-11-04T05:07:25.680+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-04T05:07:25.689+0000] {pivoting_data.py:23} INFO - Data loaded from /opt/***/dags/DataPreprocessing/src/data_store_pkl_files/test_data/test_data.pkl
[2024-11-04T05:07:25.689+0000] {pivoting_data.py:35} INFO - All necessary columns are present.
[2024-11-04T05:07:25.693+0000] {pivoting_data.py:62} INFO - Date column processed successfully.
[2024-11-04T05:07:25.697+0000] {pivoting_data.py:70} INFO - Data pivoted successfully.
[2024-11-04T05:07:25.698+0000] {pivoting_data.py:76} INFO - Pivoted DataFrame saved as '/opt/***/dags/DataPreprocessing/src/data_store_pkl_files/test_data/pivoted_test_data.pkl'.
[2024-11-04T05:07:25.698+0000] {pivoting_data.py:101} INFO - Data processing and pivoting completed successfully.
[2024-11-04T05:07:25.698+0000] {python.py:237} INFO - Done. Returned value was: []
[2024-11-04T05:07:25.698+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-04T05:07:25.706+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=datapipeline_new, task_id=pivot_data_test, run_id=manual__2024-11-04T05:06:12.986905+00:00, execution_date=20241104T050612, start_date=20241104T050725, end_date=20241104T050725
[2024-11-04T05:07:25.730+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-04T05:07:25.743+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-04T05:07:25.744+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
